src/main/java/core/basesyntax/TypeSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class TypeSupplier {
+    public String getRandomType() {
```

Comment:
```suggestion
    public FigureType getRandomType() {
```
тоді краще повертати не стрінгу а enum і потім ним оперувати в switch case